src/centos/macros.tpl

```diff
@@ -41,6 +41,19 @@ RUN \
 {% else %}
     virtualenv-$PYTHON_VERSION ${APP_ROOT} && \
 {% endif %}
+{% if spec.version != "2.7" %}
+    # Python 3 only code, Python 2 installs pip from PyPI in the assemble script. \
+    # We have to upgrade pip to a newer verison because: \
+    # * pip < 9 does not support different packages' versions for Python 2/3 \
+    # * pip < 19.3 does not support manylinux2014 wheels. Only manylinux2014 wheels \
```

Comment:
Future proofing: `Only manylinux2014 (and later) wheels` ... there's probably gonna be a new manylinux standard.

Reply:
Fixed everywhere.